Cargo.toml

```diff
@@ -16,6 +16,7 @@ default-members = [
     "quadrature",
     "tools",
     "tree",
+    "hyksort",
```

Comment:
I'm not sure if hyksort was intentionally left out here, but as it wasn't included, none of its test were running on the CI